kurlkinds/config/crds/v1beta1/cluster.kurl.sh_installers.yaml

```diff
@@ -1704,10 +1708,6 @@ spec:
                     type: string
                   noProxy:
                     type: boolean
-                  preflightIgnore:
```

Comment:
Why were these removed?

Reply:
The code generator wasn't run the last time these were changed, so that's why they are showing up in this PR.

Turns out it was my fault on the previous work for host preflights, there were renamed to the flags above: https://github.com/replicatedhq/kurl/blob/60b7cd77f2fbf869becb7b780c8f639c68a1e0ee/kurlkinds/pkg/apis/cluster/v1beta1/installer_types.go#L123-L126

Reply:
gotchya cool thanks!